sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapi/AADOboClientRegistrationRepository.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.aad.webapi;
+
+import org.jetbrains.annotations.NotNull;
+import org.springframework.security.oauth2.client.registration.ClientRegistration;
+import org.springframework.security.oauth2.client.registration.ClientRegistrationRepository;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * ClientRegistration Repository for obo flow
+ */
+public class AADOboClientRegistrationRepository implements ClientRegistrationRepository, Iterable<ClientRegistration> {
```

Comment:
Can we use `InMemoryClientRegistrationRepository` instead of create `AADOboClientRegistrationRepository`?

Reply:
@wujack778  I think we could use `InMemoryClientRegistrationRepository` directly since we don't have special logic in the client registration repo here.